contrib/gorm.io/gorm.v1/gorm.go

```diff
@@ -120,6 +120,14 @@ func after(db *gorm.DB, operationName string, cfg *config) {
 		opts = append(opts, tracer.Tag(ext.EventSampleRate, cfg.analyticsRate))
 	}
 
+	if cfg.tagFns != nil {
+		for key, tagFn := range cfg.tagFns {
+			if tagFn != nil {
```

Comment:
You are already checking `nil` in `option.go`. Is it necessary here too?

Reply:
it still is in case `WithCustomTag` is never invoked.
but I do prefer creating an empty map as default config to avoid this `nil` check.